### ISSUE TITLE
📊 Statistic, Probability and Hypothesis Testing

### ISSUE THREAD

**Issue by CSFelix** [CSFelix/data-science-studies#4]

**Comment by CSFelix** [comment]
* [ ] Types of Variables: Categorical (Ordinal and Nominal), Numeric (Continuous and Discrete), Spatial, Datatime and Sequential Texts;

* [ ] Categorical Features Statistic I: Frequence and Cross Tables (One Dimensional, Two Dimensional and Higher Dimensional), Mode and Likert Scale

* [ ] Numerical Features Statistic I - Position Metrics: Mode, Mean and Median (show them on histograms); Quartiles, Min and Max Values (show them on box-and-whisker plots)

* [ ] Numerical Features Statistic II - Dispersion Metrics: Variance, Standard Deviation, Mean Absolute Deviation, Coefficient of Variation, Skweness and Kurtosis

* [ ] Numerical Features Statistic III - Probability Distributions: Uniform, Bernoulli, Binomial, Geometric and Exponential, Poisson, Normal; Central Limit Theorem and Standard Error of the Mean (SEM)

* [ ] Hypothesis Testing I: P-Value, Confidence Level and Significance Level; Null Hypothesis (H0) and Alternative Hypothesis (H1); Parametric and Non-Parametric Tests; Bicaudal and Unicaudal Tests (check notebook where I wrote `GOKU` with a red pen 😐); Normalizer vs Standardizer vs Scales (https://datascience.stackexchange.com/questions/115313/what-is-the-difference-between-minmaxscaler-and-normalizer-in-scikit-learn and https://stackoverflow.com/questions/67940110/difference-between-normalizer-and-minmaxscaler)

* [ ] Hypothesis Testing II: Checking out for Normality - Kolmogorov-Smirnov (KS) and Shapiro Tests; Variance Homogeneity - F-Levene

* [ ] Hypothesis Testing III: Comparing Means - T-Test, Z-Test and Freedom Degrees Explanations; One Sample T-Test, Two Samples T-Test (Independent and Stacked); One Sample Z-Test, Two Samples Z-Test (Independent and Stacked)**

* [ ] Hypothesis Testing IV: Comparing Variances - ANOVAM; Measurement Metrics - MAE, MSE, RMSE, RMSLE, Huber Loss Function (https://csfelix.vercel.app/intro-to-deep-learning-loss-functions-optimizers-underfitting-overfitting-dropout-batchnormalization), SST, SSE, SSM and R²

* [ ] Bonus: pandas profiling, lux api, word cloud, data prep library, autoviz;
